lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -795,6 +796,65 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
     """
   end
 
+  @doc """
+  Chart component
+
+  You can see it in use in Metrics page.
+  """
+
+  attr :id, :string,
+    required: true,
+    doc: "Because is a stateful `Phoenix.LiveComponent` an unique id is needed."
+
+  attr :data, :list,
+    default: [],
+    doc: """
+    Temporary list of points to show in the chart.
+    Each element should be the format `{optional_label, x, y}`.
+    New points can be added using the function `send_data_to_chart/2` in real time.
+    """
```

Comment:
I don't think it makes sense to change this data attribute from temporary assign to the new `stream` feature, but maybe you deprecate temporary assigns so we should do it.